### DISCUSSION TITLE
keeping the post window out of the buffer list

### DISCUSSION THREAD

**Discussion by daveriedstra** [davidgranstrom/scnvim#229] [Ideas]
Here's a lua snippet to keep the post window out of the buffer list. Useful if you like to navigate through multiple files with `:bn` / `:bp`.

```lua
-- keep post window out of buffer list
vim.api.nvim_create_autocmd('filetype', {
    group = 'sclang',
    pattern = { 'scnvim' },
    callback = function()
        vim.api.nvim_set_option_value('buflisted', false, {scope = 'local'})
    end
})
```

**Comment by salkin-mada** [comment]
For me this only works if group is not set or group is set to `scnvim_editor`

**Reply by daveriedstra** [reply]
Good catch. I think this might have to do with whether or not the augroup exists (eg, with `vim.api.nvim_create_augroup('sclang', {})`). I'm reusing the `sclang` augroup that I use for autocmds relating to supercollider-h4x, treesitter, and some other things.
